pysynchronizer/utils.py

```diff
@@ -0,0 +1,38 @@
+
+import os
+
+from uuid import UUID
+
+def is_valid_uuid(uuid_to_test):
+    try:
+        uuid_obj = UUID(uuid_to_test)
+    except ValueError:
+        return False
+    return str(uuid_obj) == uuid_to_test
+
+def generate_uuid():
+    return str(uuid.uuid4())
+
+def column_print(rows):
+    col_width = []
+    for row in rows:
+        for idx, col in enumerate(row):
+            if idx == len(col_width):
+                col_width.append(0)
+            col_width[idx] = max(col_width[idx], len(col) + 2)
+
+    for row in rows:
+        print("".join(col.ljust(col_width[idx]) for idx, col in enumerate(row)))
+
+def path_to_uuid(path):
```

Comment:
dbus_path_to_uuid makes it a little clearer what the types are.

Reply:
yep and a bit wordier too, but i can make the change